### PR TITLE
Adds scripts for getting each stat

### PR DESCRIPTION
**Pull request by Drummerboy444**

### CODE REVIEW

**Comment by Drummerboy444** [comment]
[Scripts for accessing current stats](https://trello.com/c/SrNbCK8J/2-scripts-for-accessing-current-stats)

File: scripts/StatsManager_getHealth/StatsManager_getHealth.gml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/// @function StatsManager_getHealth(id_abilityCaster) Checks if the given instance is an ability caster, if so
```
Comment (pipkin94) [inline scripts/StatsManager_getHealth/StatsManager_getHealth.gml]: I believe the naming convention for scripts would be StatsManager_GetHealth to be consistent.

File: scripts/StatsManager_getSpeed/StatsManager_getSpeed.gml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/// @function StatsManager_getSpeed(id_abilityCaster) Checks if the given instance is an ability caster, if so
```
Comment (pipkin94) [inline scripts/StatsManager_getSpeed/StatsManager_getSpeed.gml]: Same as above with capitalisation, StatsManager_GetSpeed.

File: scripts/Player_Move/Player_Move.gml [hunk 3]
```diff
@@ -22,7 +22,7 @@ var _bool_isMovingVertically = _bool_up xor _bool_down;
 var _num_horizontalMovement = _bool_right - _bool_left;
 var _num_verticalMovement = _bool_down - _bool_up;
 
-var _num_movementSpeed = oPlayer.map_currentStats[? Enum_Stats.SPEED];
+var _num_movementSpeed = StatsManager_getSpeed(oPlayer.id);
```
Comment (pipkin94) [inline scripts/Player_Move/Player_Move.gml]: Remember to update this script use after editing below.

File: scripts/StatsManager_getHealth/StatsManager_getHealth.gml (hunk 1 above)
Comment (Drummerboy444) [reply scripts/StatsManager_getHealth/StatsManager_getHealth.gml]: Ah well spotted, thanks man! I've changed this now

File: scripts/StatsManager_getSpeed/StatsManager_getSpeed.gml (hunk 2 above)
Comment (Drummerboy444) [reply scripts/StatsManager_getSpeed/StatsManager_getSpeed.gml]: Changed here too

File: scripts/Player_Move/Player_Move.gml (hunk 3 above)
Comment (Drummerboy444) [reply scripts/Player_Move/Player_Move.gml]: Changed here too

**Pull request merged**